src/tests/match-requests.spec.ts

```diff
@@ -115,7 +118,43 @@ describe("Match requests", () => {
         key3: "c"
       },
       field2: "d",
-      field3: 1
+      field3: 1,
+      requestCount: 2,
+    });
+  });
+
+  describe('When two requests are the same', () => {
+    // @ts-ignore
```

Comment:
What is this for?

Reply:
Seems like my IDE added this. Will remove.